### PR TITLE
GM button: toggle vehicle blockers

### PR DESCRIPTION
**Pull request by Doubleumc**
# About the pull request

A convenient button to enable/disable those invisible vehicle blockers.

image
image

# Explain why it's good for the game

Significantly more convenient than the existing methods.

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl:
add: gm button to toggle vehicle blockers
/:cl:

### CODE REVIEW

**Review by morrowwolf — CHANGES_REQUESTED** [review]
Instead of having it be a mode toggle I'd far prefer a global list of all vehicle blockers that have an active var that we toggle on/off.

We're confined to a certain number of bitflags and we're starting to walk closer to the limit.

Secondly, and I'm happy to do this if you don't get to it, could you add another section that's just "flags" or "toggles". Should be just changing the categories of this and the other one to "Game Master.Toggles"

**Comment by Doubleumc** [comment]
Changed it to work like "Toggle LZ Weeding"/"Toggle OB Spawn" where it toggles a global variable that the effected system checks.

Moved "Toggle Vehicle Blockers" and "Toggle Xeno Spawning" to the Flags section to mirror the Admin tab.

File: code/modules/admin/admin_verbs.dm [hunk 1]
```diff
@@ -329,6 +329,7 @@ var/list/roundstart_mod_verbs = list(
 		add_verb(src, /client/proc/toggle_game_master)
 		add_verb(src, /client/proc/toggle_join_xeno)
 		add_verb(src, /client/proc/game_master_rename_platoon)
+		add_verb(src, /client/proc/toggle_vehicle_blockers)
```
Comment (morrowwolf) [inline code/modules/admin/admin_verbs.dm:332]: Also needs removed line 364 for de-admin'ing and such

**Review by morrowwolf — DISMISSED** [review]
Otherwise fine

File: code/modules/admin/admin_verbs.dm (hunk 1 above)
Comment (Doubleumc) [reply code/modules/admin/admin_verbs.dm:332]: Done.

**Pull request merged**